error: the name `a` is defined multiple times
  ┌─ ./test/compile-fail/resolution/conflicts/fn/top.rhdl:1:4
  │
1 │ fn a() {}
  │    - previous definition of the name `a` here
2 │ fn a() {}
  │    ^ `a` redefined here

